submariner-operator/move-subctl-to-its-own-repo.md

```diff
@@ -0,0 +1,55 @@
+# Move subctl to its own repository
+
+## Summary
+
+Currently, `subctl` code lies in `submariner-operator` repository. Both subctl and operator require different set of dependencies which
+might conflict will improving subctl/operator. For eg, subctl can't be go install-ed currently because the operator dependencies don't allow
```

Comment:
```suggestion
might conflict with improving either or both. For example, `subctl` can't be `go install`-ed currently because the operator dependencies don't allow
```